build.gradle

```diff
@@ -4,51 +4,57 @@ buildscript {
-        android_version_name = "0.1.0"
-        android_min_sdk_version = 19
+        android_version_code = 9
+        android_version_name = "0.7.0"
+        android_min_sdk_version = 21
         android_target_sdk_version = 28
         android_compile_sdk_version = 28
 
         // Dependencies (Alphabet sorted)
-        android_core_version        = '1.2.0'
+        android_core_version = '1.2.0'
+        android_fragment_version = '1.2.4'
         android_legacy_support_version = '1.0.0'
-        android_support_version     = '1.1.0'
-        android_ktx_version         = '1.0.1'
-        fragment_ktx_version        = '1.2.0'
-        android_material            = '1.1.0-alpha02'
-        arch_component_version      = '2.2.0-rc02'
+        android_lifecycle_version = '2.2.0'
+        android_material_version = '1.2.0-alpha06'
+        android_support_version = '1.1.0'
 
-        constraint_layout_version   = '2.0.0-alpha3'
+        constraint_layout_version = '2.0.0-alpha3'
+        crashlytics_version = "2.10.1"
 
-        glide_version               = '4.8.0'
-        gson_version                = '2.4'
+        firebase_version = "17.2.2"
+        firebase_perf_version = "19.0.5"
 
-        hilt_version                = '2.28-alpha'
-        hilt_androidx_version       = '1.0.0-alpha01'
+        glide_version = '4.8.0'
+        gson_version = '2.4'
 
-        kotlin_version              = '1.3.61'
+        hilt_version = '2.28-alpha'
+        hilt_androidx_version = '1.0.0-alpha01'
 
-        moshi_version               = '1.6.0'
-        multidex_version            = '2.0.1'
+        kotlin_version = '1.3.72'
 
-        okhttp_version              = '3.7.0'
+        multidex_version = '2.0.1'
+
+        navigation_version = '2.2.0-rc04'
+
+        okhttp_version = '3.7.0'
 
         permission_dispatcher_version = '4.3.0'
-        play_service_version        = '12.0.1'
-
-        retrofit_version            = '2.4.0'
-        rxandroid_version           = '2.0.1'
-        rxjava_version              = '2.1.4'
-        rxbinding_version           = '2.1.1'
-        rxkotlin_version            = '2.2.0'
-
-        test_junit_version          = '4.12'
-        test_kluent_version         = '1.36'
-        test_mockito_version        = '0.3.1'
-        test_runner_version         = '1.1.1'
-        test_robolectric_version    = '4.3.1'
-        timber_log_version          = '3.1.0'
+        play_service_version = '12.0.1'
+
+        retrofit_version = '2.4.0'
+        rxandroid_version = '2.0.1'
+        rxbinding_version = "3.1.0"
+        rxjava_version = '2.1.4'
+        rxkotlin_version = '2.2.0'
+
+        android_test_version = '1.2.0'
+        android_test_junit_version = '1.1.1'
+        test_espresso_version = "3.1.0"
+        test_junit_version = '4.13'
+        test_kluent_version = '1.60'
+        test_mockito_version = '2.1.0'
+        test_robolectric_version = '4.3.1'
+        timber_log_version = '4.7.1'
```

Comment:
Do we want to use this style, not align the `=` ?
If we want the old format I think we might need to rearrange it for align `=` in every line.

Reply:
@lydiasama yeah, we should use the old style, I will update it 💪

Reply:
@lydiasama @sleepylee @manh-t @hoangnguyen92dn @Tuubz  I would not agree to this special formatting: `align all dependency versions on the same vertical column on the right`, because:
- We don't touch on this dependency versions config too much, even in case we keep the format without aligning `=` we still able to determine versions easily for upgrading or downgrading... not so hard from my view tbh.
- Applying this special formatting would make hard for us to maintenance, especially for new members to understand what should they do when modifying this file. I know we could create a special style for Groovy file type to keep this format but does that make sense for us to add 1 more step in our process?

I personally only agree on which action helps us to speed up and make easy our development process, not just to keep our code beautiful but brings unnecessary extra steps to achieve. Please bring your ideas up here 🙏